src/app/@ansyn/ol/maps/open-layers-map/openlayers-map/openlayers-map.ts

```diff
@@ -351,6 +354,8 @@ export class OpenLayersMap extends BaseImageryMap<OLMap> {
 		olShare.removeWorkers(layer);
 		this._mapLayers = this._mapLayers.filter((mapLayer) => mapLayer !== layer);
 		this._mapObject.removeLayer(layer);
+		// TODO: Should we remove the layer from the background map too?
```

Comment:
no because we only add to it the main layer and we clear the background Map before new layer is added